### ISSUE TITLE
RGB Composition

### ISSUE THREAD

**Issue by pcampanella** [fadeoutsoftware/WASDI#13]
Understand the ViewPort mechanism
Try to solve the JPG transparency problem
add the new operation to the launcher.

**Comment by pcampanella** [comment]
I made some test on our code.
In the BandImageManager there is a buildRGBImage version able to create an image of the requested output dimension. It takes more or less 160 sec for Sentinel 2 Image.
In the ProofBandImage test  and the new Test_RGB_Only test file the method is called with a S2 image and the output saved in JPG and PNG.

Total time for saving JPG is + 6 sec, PNG +30 sec.
Outuput JPG size 20Mb, PNG 250Mb.

The problem is that JPG is opaque. I posted a thread on the Step User Forum:
https://forum.step.esa.int/t/export-rgb-image-by-code-jpeg-opaque-output/13449

Hope they will answer us.

**Comment by pcampanella** [comment]
We got the answer from the Forum. Confirmed that JPG does not support transparency. They suggested another way to create a JPG but is very slow.

We need to talk about this. A solution can be:
.Add to the launcher the possibility to create and RGB image from 3 generic bands. This will create a PNG. We can evaluate a post generation conversion between PNG and JPEG, ie with GDAL (to be tested both for performance and final result)
.From the client give to the user the possibility to create a RGB. This will trigger the launcher. Once the image is done... donwload it? Save it on the server so that can be shown in the Image Editor? Both solutions?

.When we import in WASDI a new image, we can schedule a job to generate the classic RGB View and store it (again, store in JPG beacuase there is a huge space difference). This RGB image can be associated to the product itself, ie open it when in the editor clicks on the image (and not on a single band) and/or use it to generate thunmbail previews
